__tests__/middleware-test.js

```diff
@@ -0,0 +1,158 @@
+jest.mock('@agoric/casting', () => {
+    return {
+        iterateReverse: jest.fn(),
+        makeLeader: jest.fn(),
+        makeFollower: jest.fn(),
+    };
+});
+
+jest.mock('../src/lib/rpc.js', () => {
+    return {
+        boardSlottingMarshaller: jest.fn(),
+        networkConfig: { rpcAddrs: ["http://127.0.0.1:26657"], chainName: "agoriclocal" },
```

Comment:
exceeds 80 char limit

Reply:
Fixed in https://github.com/jacquesvcritien/agoric-cl-middleware/commit/e7d6c899fec472591f9ed9fa8dfad2c660fbb9a5